### ISSUE TITLE
Error with a git dependency

### ISSUE THREAD

**Issue by tcitworld** [Cantido/hex_licenses#2]
I have a dependency that's registered from github, something like this:
```elixir
{:icalendar, github: "tcitworld/icalendar"},
```

The output is the following as there's no `hex_metadata.config` file for those are these dependencies are not in hex.pm.
For now they can be simply ignored by matching this case.

```
** (FunctionClauseError) no function clause matching in anonymous fn/1 in HexLicenses.license_check/1    
    
    The following arguments were given to anonymous fn/1 in HexLicenses.license_check/1:
    
        # 1
        {:icalendar, {:error, :enoent}}
    
    lib/hex_licenses.ex:32: anonymous fn/1 in HexLicenses.license_check/1
    (elixir 1.12.1) lib/enum.ex:1557: anonymous fn/3 in Enum.map/2
    (elixir 1.12.1) lib/enum.ex:3923: anonymous fn/3 in Enum.map/2
    (elixir 1.12.1) lib/stream.ex:1707: anonymous fn/3 in Enumerable.Stream.reduce/3
    (elixir 1.12.1) lib/task/supervised.ex:393: Task.Supervised.stream_deliver/7
    (elixir 1.12.1) lib/stream.ex:1719: Enumerable.Stream.do_each/4
    (elixir 1.12.1) lib/enum.ex:3923: Enum.map/2
    lib/hex_licenses.ex:32: HexLicenses.license_check/1
```

**Comment by Cantido** [comment]
Thank you for telling me about this! I thought I had some logic to handle deps not in hex but I must have broken that.

**Comment by Cantido** [comment]
I just published [v0.3.2](https://github.com/Cantido/hex_licenses/releases/tag/v0.3.2) which should fix this issue for you. Thanks again for reporting it!

**Comment by tcitworld** [comment]
Works perfectly, thanks!
